src/utils/NetlogoWEB.py

```diff
@@ -0,0 +1,272 @@
+
+# next step is putting on the refrences on the page
+# getting the data looping thru each of the years and years as heading.
+# in press, boolean top, section.
+
+from directus_sdk_py import DirectusClient
+from bs4 import BeautifulSoup
+import os
+
+# Directus setup (Replace with the correct API URL)
+DIRECTUS_URL = "https://backend.netlogo.org"  # Base API URL, not Admin panel URL
+DIRECTUS_TOKEN = "0ylVAJfZwVbnR3YKdSlbkA8pwGIhU4Ca"
+
+
+
+# Initialize Directus client
+client = DirectusClient(DIRECTUS_URL, DIRECTUS_TOKEN)
+
+def extract_references(file_path):
+    with open(file_path, encoding="utf8") as file:
+        soup = BeautifulSoup(file, 'html.parser')
+
+    References = []
```

Comment:
Change `References` to `references` or `refs`. This is just a style thing, but standard python style is that variables are all lower case (unless they are global in which they are all upper case). Having just the first letter capitalized is usually just used for class names.